app/src/main/java/ch/blinkenlights/android/vanilla/CoverCache.java

```diff
@@ -463,6 +468,19 @@ public Bitmap createBitmap(Song song, long maxPxCount) {
 					}
 				}
 
+				if (inputStream == null && (CoverCache.mCoverLoadMode & CoverCache.COVER_MODE_INLINE) != 0)
+				{
```

Comment:
please keep the formatting style consistent: the curly bracket should not be on a single line.